app/main.py

```diff
@@ -1,3 +1,16 @@
 class OnlineCourse:
-    # write your code here
-    pass
+    def __init__(self, name, description, weeks):
+        self.name = name
+        self.description = description
+        self.weeks = weeks
+
+    @staticmethod
+    def days_to_weeks(days):
+        return days // 7 if days % 7 == 0 else (days // 7 + 1)
+
+    @classmethod
+    def from_dict(cls, course_dict):
+        name = course_dict["name"]
+        description = course_dict["description"]
+        week = cls.days_to_weeks(course_dict["days"])
+        return OnlineCourse(name, description, week)
```

Comment:
```suggestion
        return cls(name, description, week)
```
Use `cls` in classmethods instead of the name of the class. It is important in the case of inheritance which will be discussed later.